src/templates/varTemplates.ts

```diff
@@ -1,16 +1,38 @@
 import * as ts from 'typescript'
+import * as vsc from 'vscode'
+import * as _ from 'lodash'
+import { getVariableNameFromCallExpresion } from 'const-name'
 import { CompletionItemBuilder } from '../completionItemBuilder'
 import { BaseExpressionTemplate } from './baseTemplates'
+import { clipByLastIndex } from 'const-name/build/util'

```

Comment:
Isn't that little fishy to import internal functions from a lib like that?

If it was supposed to be exported then I guess it should be officially exported so that one doesn't have to use `lib-name/build/*`.
I'm not an expert with npm libs but it's the first time I see something like this.

Reply:
Anyway, I removed this import in last commit. Don't need to reuse internal things here.